cis/schema.json

```diff
@@ -233,15 +243,15 @@
                     "lastUsed": {
                         "type": "string",
                         "format": "date-time",
-                        "description": "UNKNOWN"
+                        "description": "Date at which this group membership last granted access to a resource (generally, an RP) for the user. This may be used to automatically expire and deny access when the lastUsed attribute is older than 3 month, for example."
                         },
                     "name": {
                         "type": "string",
-                        "description": "Name of the group, as given by the IdP."
+                        "description": "Name of the group."
                         },
                     "uuid": {
                         "type": "string",
-                        "description": "Reserved; profile providers should not use this value."
+                        "description": "Unique identifier for the authoritative group. Profile consumers should not try to interpret this value in any way."
```

Comment:
if you wanted to use these groups, then yes. In most cases you'll want to use `groups` though.

There were many questions about `groups` vs `authoritativeGroups` and ideally I/we should make a little doc about it (descriptions are a little short to tell the whole story).

Another attempt through comments though:

- Auth0 (the `access provider`) is expected to maintain and enforce `AuthoritativeGroups`. This means it may prevent a user to login to an RP, regardless of the RP's opinion on this. While **all** RPs have an associated `AuthoritativeGroups` attribute, not all of them prevent access (basically any RP set to "allow anyone in" does not prevent access, it just updates attributes)
- RPs *may* use `AuthoritativeGroups` values if they wish, though Auth0 already did the access validation on their behalf
- RPs *should* use `groups` (any, or combination of groups, as the RP sees fit) with full knowledge that the groups are only as reliable as the people managing them. eg:

  - if there is a group called "mozilliansorg_pplwholikechocolate" managed by a person in Japan that maintain a directory of friends who like chococolate and you decide to use it, it's only as reliable as what this person decides the members are.
  - If you create your **own** "mozilliansorg_pplthatitrust" (mozillians.org) or "pplthatitrust" (LDAP manage) then it's up to you.
  - If you use "workday_staff", these are managed by Mozilla HR, though they currently do not have the same guarantees as `AuthoritativeGroups`, but they hopefully and eventually will. If we get there, there will most likely be a "publisher" attribute in the `AuthoritaiveGroups` (adding being much easier than removing/changing attributes)